doc/users/whats_new/updated_backend_wx.rst

```diff
@@ -0,0 +1,12 @@
+wx backend has been updated
+---------------------------
+The wx backend can now be used with both wxPython classic and
+`Phoenix <http://wxpython.org/Phoenix/docs/html/main.html>`__.
+
+wxPython classic has to be at least version 2.8.12 and works on Python 2.x,
+wxPython Phoenix needs a current snapshot and works on Python 2.7 and 3.4+.
```

Comment:
"needs a current snapshot" <-- this phrase will not age well. I would consider taking it out, and just stating what versions of python it is intended for (which is entirely the point of the sentence). This version of matplotlib won't be released for a few more months, anyway.